apps/hermes/client/js/src/index.ts

```diff
@@ -0,0 +1,18 @@
+export {
```

Comment:
Might as well just get rid of this file now and move `HermesClient.ts` to `index.ts` or just point the package.json at `HermesClient.ts` instead.  This file serves no purpose and just adds indirection.